lib/cfn-nag/custom_rules/SecurityGroupRuleDescriptionRule.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'cfn-nag/violation'
+require_relative 'base'
+require 'cfn-nag/ip_addr'
+
+class SecurityGroupRuleDescriptionRule < BaseRule
+  def rule_text
+    'Security group rules without a description obscure their purpose and may '\
+    'lead to bad practices in ensuring they only allow traffic from the ports '\
+    'and sources/destinations required.'
+  end
+
+  def rule_type
+    Violation::WARNING
+  end
+
+  def rule_id
+    'W36'
+  end
+
+  def violating_sg_part(sg_part)
```

Comment:
make all of these violating* methods private

Reply:
fixed